### DISCUSSION TITLE
Adding power control support for G15 2020 GA502 models

### DISCUSSION THREAD

**Discussion by Marcopologist** [seerge/g-helper#138] [Ideas]
Hey everyone! Just wanted to say congrats on the development of this app. I don't have much expertise with development but I have to say I haven't had any major issues with the app on my GA502 G15. The only issue is that I don't have control of power limits and it may be causing my laptop to drain more power when plugged in through the 65w usb-c port causing blue screens. So if there is a way to add support for this I am happy to get involved in the development (should you need any logs from my end). Thanks!

**Comment by seerge** [comment]
Hello. There is a topic where volunteers look for endpoints that could  control PPTs on older models https://github.com/seerge/g-helper/issues/38 . If you want - you can check it too. 

But in general - it could be that your laptop doesn't support power limits at all (at least via asus wmi that app uses) :)
